web/src/admin/AdminInterface/AdminInterface.ts

```diff
@@ -158,3 +158,9 @@ export class AdminInterface extends EnterpriseAwareInterface {
         ></ak-locale-context>`;
     }
 }
+
```

Comment:
Warning: Aside from the _last two_ files in this PR, this one and the next 359 after it are all exactly this.  If you want to see _how_ this was done, scroll down and read those last two.